initializr-actuator/src/main/java/io/spring/initializr/actuate/autoconfigure/InitializrActuatorEndpointsAutoConfiguration.java

```diff
@@ -30,6 +32,7 @@
  * @author Stephane Nicoll
  */
 @Configuration
+@AutoConfigureAfter(InitializrAutoConfiguration.class)
```

Comment:
Why do we need this? There isn't any condition that require this order AFAIK

Reply:
we don't 😬